examples/oqmd/oqmd.ipynb

```diff
@@ -1,670 +1,777 @@
 {
- "cells": [
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# OQMD Mini Example"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 1,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from foundry import Foundry"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# Instantiate Foundry from local file and load data from remote"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "f = Foundry()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "#need Globus to pull data\n",
-    "f = f.load('_test_foundry_oqmd_v1.1')\n",
-    "X,y = f.load_data()"
-   ]
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
```

Comment:
Does this notebook run properly, it looks like there's a reference to a function called `oqmd_to_df()` that's never defined, nor is it imported?? I have no idea what's going on here

Reply:
found the missing cell! Great catch